src/resolvers/products/index.ts

```diff
@@ -0,0 +1,49 @@
+import { generateId, EntityType } from '../../functions/generate-binary-id';
+import BinaryResolver from '../../schema/scalars/customs/binary-scalar';
+import { CreateProductInput } from '../../types/products-types';
+import ProductModel from '../../models/products';
+import { Context } from 'koa';
+import { Account } from '../../types/accounts-types';
+
+export const ProductResolver = {
+  Binary: BinaryResolver,
```

Comment:
I think it's unnecessary to add `BinaryResolver` for both of your `ProductResolver` and `AccountResolver`. Maybe you can just add something like `ScalarResolvers` and put `Binary` in there together with other scalars.